src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/search/parameter/SearchParameterKey.java

```diff
@@ -8,9 +8,10 @@ public enum SearchParameterKey {
     USER("user"),
     JURISDICTION("jurisdiction"),
     STATE("state"),
-    TASK_ID("taskId"),
-    TASK_TYPE("taskType"),
-    CASE_ID("caseId"),
+    TASK_TYPE_CAMEL_CASE("taskType"),
```

Comment:
Is anyone using this one?  If not then we should only support the `snake_case`

Reply:
good point, ExUI does not use, will check our functional tests and remove the camelcase